### DISCUSSION TITLE
how to get all text of this element i get only 1 element i cant mark multiple and get it here are elements

### DISCUSSION THREAD

**Discussion by TanMink** [AutomaApp/automa#187] [Q&A] [answered]
how to get all text of this element i get only 1 element i cant mark multiple and get it here are elements
.S-product-item:nth-child(1) .S-product-item__retail-price   
.S-product-item:nth-child(2) .S-product-item__retail-price
.S-product-item:nth-child(3) .S-product-item__retail-price
.S-product-item:nth-child(4) .S-product-item__retail-price
.S-product-item:nth-child(5) .S-product-item__retail-price

**Comment by Kholid060** [comment, marked as answer]
try `.S-product-item .S-product-item__retail-price` as the element selector

**Reply by TanMink** [reply]
if they are
```
# risultato-1_0  .title
# risultato-1_1 .title
# risultato-1_2 .title
# risultato-1_3 .title
# risultato-1_4 .title
```
 then what do we do. loops and concatenations are required
